packages/expo-auth-session/src/providers/Google.ts

```diff
@@ -167,7 +168,7 @@ class GoogleAuthRequest extends AuthRequest {
    */
   async getAuthRequestConfigAsync(): Promise<AuthRequestConfig> {
     const { extraParams = {}, ...config } = await super.getAuthRequestConfigAsync();
-    if (config.responseType === ResponseType.IdToken && !extraParams.nonce && !this.nonce) {
+    if (config.responseType.split(' ').includes(ResponseType.Token) && !extraParams.nonce && !this.nonce) {
```

Comment:
Why does this change from `IdToken` to `Token`? Seems unintentional.

Reply:
Oy. Reading one thing, typing another.